### ISSUE TITLE
Image disposed error in the repository dialog

### ISSUE THREAD

**Issue by HiromuHota** [HiromuHota/pentaho-kettle#38]
Under some condition, clicking on a "Connect" button at the top-right corner causes an "Image is disposed" error as below.
```
java.lang.IllegalArgumentException: Image is disposed
	org.eclipse.rap.rwt.remote.JsonMapping.toJson(JsonMapping.java:185)
	org.eclipse.rap.rwt.internal.lifecycle.WidgetLCAUtil.renderProperty(WidgetLCAUtil.java:424)
	org.eclipse.swt.internal.widgets.shellkit.ShellLCA.renderImage(ShellLCA.java:198)
	org.eclipse.swt.internal.widgets.shellkit.ShellLCA.renderChanges(ShellLCA.java:121)
	org.eclipse.swt.internal.widgets.shellkit.ShellLCA.renderChanges(ShellLCA.java:1)
	org.eclipse.rap.rwt.internal.lifecycle.WidgetLCA.render(WidgetLCA.java:40)
	org.eclipse.swt.internal.widgets.displaykit.DisplayLCA$RenderVisitor.render(DisplayLCA.java:313)
	org.eclipse.swt.internal.widgets.displaykit.DisplayLCA$RenderVisitor.visit(DisplayLCA.java:297)
	org.eclipse.swt.internal.widgets.WidgetTreeUtil.accept(WidgetTreeUtil.java:32)
	org.eclipse.swt.internal.widgets.displaykit.DisplayLCA.renderShells(DisplayLCA.java:175)
	org.eclipse.swt.internal.widgets.displaykit.DisplayLCA.render(DisplayLCA.java:101)
	org.eclipse.rap.rwt.internal.lifecycle.Render.execute(Render.java:32)
	org.eclipse.rap.rwt.internal.lifecycle.RWTLifeCycle.continueLifeCycle(RWTLifeCycle.java:155)
	org.eclipse.rap.rwt.internal.lifecycle.RWTLifeCycle.sleep(RWTLifeCycle.java:229)
	org.eclipse.swt.widgets.Display.sleep(Display.java:1258)
	org.pentaho.di.ui.repo.RepositoryDialog.open(RepositoryDialog.java:255)
	org.pentaho.di.ui.repo.RepositoryDialog.open(RepositoryDialog.java:85)
	org.pentaho.di.ui.repo.RepositoryDialog.openLogin(RepositoryDialog.java:278)
	org.pentaho.di.ui.repo.RepositoryConnectMenu$4$1.widgetSelected(RepositoryConnectMenu.java:177)
	org.eclipse.swt.widgets.TypedListener.handleEvent(TypedListener.java:263)
	org.eclipse.swt.widgets.EventTable.sendEvent(EventTable.java:109)
	org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:687)
	org.eclipse.swt.widgets.Widget.notifyListeners(Widget.java:594)
	org.eclipse.swt.widgets.Display.executeNextEvent(Display.java:1217)
	org.eclipse.swt.widgets.Display.runPendingMessages(Display.java:1198)
	org.eclipse.swt.widgets.Display.safeReadAndDispatch(Display.java:1181)
	org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:1173)
	org.eclipse.rap.rwt.application.AbstractEntryPoint.createUI(AbstractEntryPoint.java:69)
	org.eclipse.rap.rwt.internal.lifecycle.RWTLifeCycle.createUI(RWTLifeCycle.java:177)
	org.eclipse.rap.rwt.internal.lifecycle.RWTLifeCycle$UIThreadController.run(RWTLifeCycle.java:290)
	java.lang.Thread.run(Thread.java:745)
	org.eclipse.rap.rwt.internal.lifecycle.UIThread.run(UIThread.java:107)
```

**Comment by HiromuHota** [comment]
It turned out to be another multi-session issue.

**Comment by HiromuHota** [comment]
Resolved by a13b04085aff3eea469075b14fa8c098fcca8457.
